### PR TITLE
Added how to use labels to break nested loops to trpl.

### PR DESCRIPTION
**Pull request by echochamber**
Was browsing somebody else's code and came across a snippet using labels. Looking around, it seems like there was an example for this in [rustbyexample](http://rustbyexample.com/flow_control/loop/nested.html) but none in trpl.

### CODE REVIEW

File: src/doc/trpl/while-loops.md [hunk 1]
```diff
@@ -88,6 +88,18 @@ for x in 0..10 {
 }
 ```
 
+You may also encounter situations where you have nested loops and need to specify which one your `break` or `continue` statement is for. Like most other languages, by default a `break` or `continue` will apply to innermost loop. In a sitation where you would like to a `break` or `continue` for one of the outer loops, you can use labels to specify which loop the `break` or `continue` statement applies to. This will only print when both x and y are odd:
```
Comment (steveklabnik) [inline src/doc/trpl/while-loops.md]: can you wrap this line please? Also, `x` and `y` need to be in graves for highlighting

File: src/doc/trpl/while-loops.md [hunk 2]
```diff
@@ -88,6 +88,18 @@ for x in 0..10 {
...
+
+```rust
+'outer: for x in 0..10 {
+    'inner: for y in 0..10 {
+        if x % 2 == 0 { continue 'outer; } // Continues the loop over x
```
Comment (steveklabnik) [inline src/doc/trpl/while-loops.md]: these two comments should have either a period at the end, or a lower case `C`

**Comment by steveklabnik** [comment]
Thanks! two nits and then r=me

**Comment by echochamber** [comment]
?r @steveklabnik 

Updated with fixes for your comments.

**Comment by echochamber** [comment]
r? @steveklabnik 

Updated with fixes for your comments.

**Comment by steveklabnik** [comment]
Thank you!

**Pull request merged**